### DISCUSSION TITLE
EJBCA Community 7.4.3.2.

### DISCUSSION THREAD

**Discussion by ramiabdallah1512** [Keyfactor/ejbca-ce#536] [Q&A]
Hello,
I am currently using EJBCA Community 7.4.3.2. I would like to know if this version is still valid and if its lifetime is still acceptable. Two vulnerabilities have been detected in the meantime. Are they resolved in this version, or do I need to upgrade?

**Comment by primetomas** [comment]
This release if from december 2020, so more than 3 years old. 
https://doc.primekey.com/ejbca/ejbca-release-information/ejbca-release-notes/ejbca-change-log-summary
You should never rely on such old software. You need to upgrade.
